### DISCUSSION TITLE
JQuery version in built site

### DISCUSSION THREAD

**Discussion by emh1908** [squidfunk/mkdocs-material#3664] [Q&A] [answered]
(Apologies as I'm not technical). I've been advised that the jQuery version for the built site (showing as jquery-3.3.1.min.js) needs to be upgraded to the latest jQuery version. Is this something I can do?

**Comment by squidfunk** [comment, marked as answer]
Material for MkDocs does not use jQuery.

**Reply by emh1908** [reply]
Ah OK - thank you.
